base/src/main/java/io/spine/validate/rules/ValidationRules.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.validate.rules;
+
+import com.google.common.collect.ImmutableCollection;
+import com.google.common.collect.ImmutableSet;
+import io.spine.option.ValidationTargetsParser;
+
+import java.util.Collection;
+import java.util.Properties;
+
+import static io.spine.util.PropertyFiles.loadAllProperties;
+
+/**
+ * Utilities for obtaining {@linkplain ValidationRule validation rules} known for the application.
+ *
+ * <p>During initialization of this class, definitions of validation rules are
+ * {@linkplain Builder#put(Properties) validated}. If an invalid validation rule was found,
+ * a runtime exception will be raised.
+ *
+ * @author Dmytro Grankin
+ */
+public class ValidationRules {
+
+    /**
+     * A name of the file, which contains validation rules and their target fields paths.
+     */
+    private static final String VAL_RULES_PROPS_FILE_NAME = "validation_rules.properties";
+
+    private static final ImmutableCollection<ValidationRule> rules = new Builder().build();
+
+    private ValidationRules() {
+        // Prevent instantiation of this class.
+    }
+
+    /**
+     * Obtains validation rules known to the application.
+     *
+     * @return the immutable collection of validation rules
+     */
+    @SuppressWarnings("ReturnOfCollectionOrArrayField") // It's ok to return
+                                                        // an immutable collection.
+    static ImmutableCollection<ValidationRule> getRules() {
+        return rules;
+    }
+
+    public static String getValRulesPropsFileName() {
+        return VAL_RULES_PROPS_FILE_NAME;
+    }
+
+    /**
+     * {@code Builder} assembles the validation rules from the
+     * {@linkplain #VAL_RULES_PROPS_FILE_NAME properties} files.
```

Comment:
Probably, `file` (singular). Or, if plural, please append something about classpath scan — most likely that would be a part of the string constant description.